src/layers.scss

```diff
@@ -0,0 +1,4 @@
+// Establish CSS cascade layers.
+// ⚠️ WARNING: This file must be called before other React UI styles for the cascade layers to work as intended.
+
+@layer theme, foundation, helpers, components, utilities;
```

Comment:
I realized the component styles are currently unlayered, thus ordered as last (after the `utilities` layer). It seems this is what I need to do with all component classes:

```css
.root {
    @layer components.checkboxField {
        /* root styles */
    }
}
```